RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Mock.swift

```diff
@@ -4,14 +4,15 @@ extension AccountPortfoliosClient: TestDependencyKey {
 
 	public static let testValue = AccountPortfoliosClient(
 		fetchAccountPortfolios: unimplemented("\(AccountPortfoliosClient.self).fetchAccountPortfolios"),
-		fetchAccountPortfolio: unimplemented("\(AccountPortfoliosClient.self).fetchAccountPortfolio"),
+		fetchAccountPortfolio: unimplemented("\(AccountPortfoliosClient.self).fetchAccountPortfolio"), portfolioUpdates: unimplemented("\(AccountPortfoliosClient.self).fetchAccountPortfolio"),
 		portfolioForAccount: unimplemented("\(AccountPortfoliosClient.self).portfolioForAccount"),
 		portfolios: unimplemented("\(AccountPortfoliosClient.self).portfolios")
 	)
 
 	public static let noop = AccountPortfoliosClient(
 		fetchAccountPortfolios: { _, _ in throw NoopError() },
 		fetchAccountPortfolio: { _, _ in throw NoopError() },
+		portfolioUpdates: { fatalError() },
```

Comment:
Use `AsyncLazySequence([]).eraseToAnyAsyncSequence()` instead, we should never use `fatalError` :P the rest ought to be changed to :)